### DISCUSSION TITLE
Problem facing while deploying streamlit app

### DISCUSSION THREAD

**Discussion by rinkuChhokar** [community/community#30799] [Other Feature Feedback, Questions, & Ideas]
Hello everyone,
While deploying my web app on streamlit cloud, a strange error is shown on the screen telling me-
“:exclamation: Updating the app files has failed: open /home/appuser/.ssh/id_github: no such file or directory”

the log image is here-
![Screenshot (1243)](https://user-images.githubusercontent.com/91409108/186675068-c37e445d-7e8f-43eb-abbc-aa63e59bc749.png)

I do not understand why it gives me this type of error when I don’t have these files or folders in my local app folder.

Please help me, I have to submit my project by this week…

**Comment by jfullstackdev** [comment]
fortunately they have the forum too, 

and maybe this might be of help

https://discuss.streamlit.io/t/no-such-file-or-directory-error-streamlit-cloud/21234
